automatic/logfusion/update.ps1

```diff
@@ -0,0 +1,37 @@
+﻿Import-Module AU
+
+function global:au_BeforeUpdate {
+    $Latest.ChecksumType32 = 'sha256'
+
+    Get-RemoteFiles -Purge
+
+    $file = Get-Item tools\*.exe | Select-Object -first 1
+    Remove-Item $file -Force -ErrorAction SilentlyContinue
+}
+
+function global:au_SearchReplace {
+    return @{
+        ".\tools\chocolateyInstall.ps1" = @{
+            "(?i)(^[$]installer\s*=\s*)('.*')" = "`$1'$([System.IO.Path]::GetFileName($Latest.Url32))'"
```

Comment:
Remove this replacement, you won't need it for this package as it isn't an embedded package.